ISSUE_TEMPLATE/doc_issue.md

```diff
@@ -0,0 +1,34 @@
+### Does the doc defect  apply to a code bug? 
```

Comment:
I don't really get this point of this.

Reply:
We ask that question so that we can coordinate the publishing of the doc update with the code update. Would you like the doc and code update to go out together?